docs/planning/.env.example

```
@@ -0,0 +1,14 @@
+# Database Configuration
+POSTGRES_USER=fragrance
+POSTGRES_PASSWORD=fragrance_secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)